debian/rules

```diff
@@ -182,6 +182,10 @@ endif
            PREFIX=/usr SYSCONFDIR=/etc/fossology LOCALSTATEDIR=/var \
            -C src/reportImport install
 
+    $(MAKE) DESTDIR=$(CURDIR)/debian/fossology-softwareHeritage \
```

Comment:
As the package name in `debian/control` is changed, following folder should be changed as well.

```suggestion
    $(MAKE) DESTDIR=$(CURDIR)/debian/fossology-softwareheritage \
```